### PR TITLE
✨ Add logout

### LINKED ISSUE #104: ✨ Add logout
This is a problem when we change the authentication system or the token expires and employees have to clean the cache manually

### PR DESCRIPTION
**Pull request by jsulpis**
Closes #104 

After signing in, show a logout button which:
- log out of Firebase
- log out of auth0 (clear local storage)
- redirect to the auth0 sign in page

### CODE REVIEW

File: ui/package.json [hunk 1]
```diff
@@ -25,11 +25,11 @@
     "ts-loader": "5.3.0",
     "ts-node": "7.0.1",
     "typescript": "3.1.4",
-    "webpack": "4.23.1",
-    "webpack-cli": "3.1.2"
+    "webpack": "^4.44.2",
```
Comment (hgwood) [inline ui/package.json:28]: Could you pin the exact version like it's done for other dependencies? Makes things less prone to breakage as time passes.

**Review by hgwood — COMMENTED** [review]
Thanks! Code looks OK, I'll test it tomorrow.

File: ui/package.json (hunk 1 above)
Comment (jsulpis) [reply ui/package.json]: I did not notice, it's done. FYI I have seen versions that are not pinned in all the `package.json` files. I don't know if you want to pin everything or just some packages ?

Comment (hgwood) [reply ui/package.json]: Everything :)

**Review by hgwood — APPROVED** [review]
Works nicely, thanks :)

**Pull request merged**